game/hud/src/components/DevUI/index.tsx

```diff
@@ -340,8 +370,11 @@ class DevUI extends React.PureComponent<{}, ObjectMap<RootPage> | null> {
   public render() {
     if (!this.state) return null;
     const keys = Object.keys(this.state);
+    const anyMaximized = keys.reduce((p, k) => {
+      return (this.state[k].visible && this.state[k].maximized) ? true : p;
+    }, false);
     return (
-      <Container>
+      <Container style={{ zIndex: anyMaximized ? 10000 : 999 }}>
```

Comment:
Doing this means that all DevUI's will cover the HUDWidgets (Settings, Game menu, etc.) which we don't want.

I would say just add a `DevUI` value after `Progression` to the HUDZOrder enum in game/hud/services/sessions/HUDZOrder.ts. Also, add a `MaximizedDevUI` property that goes before the GameMenu and Settings and use these values for the zIndex property.

Reply:
This is resolved, although I have not had a great deal of time to test everything is working nicely